### PR TITLE
Add new contact Form

### PR DESCRIPTION
**Pull request by RHK-MICROVERSE**
Update index.html file
Update mobile.css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -295,26 +295,55 @@ <h1 class="greetings2">About Me</h1>
...
+                  class="btn-get">Get in touch</button>
+        </li>
+      </ul>
     </form>
-    <div class="line-footer"></div>
+    <!-- Get In Touch From ENDS-->
 
+    <div class="line-footer"></div>
     <!-- Social Media-->
     <div class="social-media-footer">
```
Comment (aamirkhan2478) [inline index.html:348]: - [x] Kindly use all social media icons as mentioned in the Figma design check below :point_down: 

**Your design:**
Screenshot from 2023-02-21 23-18-05

**Figma Design:**
Screenshot from 2023-02-21 23-14-24

File: index.html [hunk 2]
```diff
@@ -295,26 +295,55 @@ <h1 class="greetings2">About Me</h1>
...
+          <!--Define Input Text Area-->
+        <li>
+          <textarea class ="textarea" 
+                    placeholder="Enter text here"
+                    name="user-message"
+                    maxlength="500"
+                     required></textarea>
+        </li>
+        <li class="button">
+          <button type="submit"
```
Comment (aamirkhan2478) [inline index.html:339]: - [x] Kindly use the same button effects as mentioned in the Figma Design check below :point_down: 
Screenshot from 2023-02-21 23-13-53

File: index.html [hunk 3]
```diff
@@ -295,26 +295,55 @@ <h1 class="greetings2">About Me</h1>
...
+      <!--Define the Full Name input-->
+      <ul class = "ul-form">
+        <li>
+          <input class="common" type="text" 
+                                placeholder="Full name" 
+                                maxlength="30"
+                                name="fullname" 
+                                required 
+                                class="Name"/>
+        </li>
```
Comment (aamirkhan2478) [inline index.html:316]: - [x] Please follow the same design provided by Figma your design is not the same as Figma's design check below :point_down: 

**Figma Design**
Screenshot from 2023-02-21 23-12-38

**Your Design**
Screenshot from 2023-02-21 23-21-54 (1)

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @RHK-MICROVERSE , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters checked

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (RHK-MICROVERSE) [reply index.html:316]: Thank you for the observation.

File: index.html (hunk 2 above)
Comment (RHK-MICROVERSE) [reply index.html:339]: Thank you for the observation.

File: index.html (hunk 1 above)
Comment (RHK-MICROVERSE) [reply index.html:348]: Thank you for the observation.

**Comment by enniopo** [comment]
Hi @RHK-MICROVERSE, this is a pair-programming activity, and you need to work on this with your coding partner. The review done by @aamirkhan2478 will be invalidated as there are no commits from your partner.

File: index.html (hunk 3 above)
Comment (souad988) [reply index.html:316]: Good job an making those changes 👍 one more thing to mention for this headline of this section is the color please make sure to use the same ones as in figma design, your  title is black and in figma #172B4D

File: index.html (hunk 2 above)
Comment (souad988) [reply index.html:339]: Kindly use the same effect as requested above in your project  you are using the hover effect design for press event

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @RHK-MICROVERSE , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212385281-4797c58a-d358-423a-aa7c-832c22203865

## Highlights
✔Good communication with reviewer 
✔Pass all linter checks 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- specifying changes done in this feature in your PR  summary because update file x  doesn't give any idea about the work .

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @RHK-MICROVERSE, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅  
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**